File: C/URI_judge_1036.c
#include <stdio.h>
#include <math.h>

int main(){
    double A,B,C,x1,x2,r1,r2,delta;
    scanf("%lf %lf %lf",&A,&B,&C);

    delta = pow(B,2) - (4*A*C);
    x1 = -B + sqrt(delta);
    r1 = x1/(2*A);
    x2 = -B - sqrt(delta);
    r2 = x2/(2*A);

    if (delta < 0){
        printf("Impossivel calcular\n");
    }
    else if (x1 == 0 || x2 == 0){
        printf("Impossivel calcular\n");
    }
    else{
        printf("R1 = %.5lf\n",r1);
        printf("R2 = %.5lf\n",r2);
    }
}

File: Python/URI_judge_1036.py
import math

bhaskara = input().split()
a,b,c = map(float,bhaskara)

d = pow(b,2) - (4*a*c)
x1 = -b +math.sqrt(d)
x2 = -b -math.sqrt(d)
r1 = x1/(2*a)
r2 = x2/(2*a)

if(d<0):
    print("Impossivel calcular")
elif(x1 == 0 or x2 == 0):
    print("Impossivel calcular")
else:
    print(f"R1 = {r1:.5f}")
    print(f"R2 = {r2:.5f}")